scripts.js

```diff
@@ -0,0 +1,80 @@
+const bookList = document.querySelector('.books');
+
+const bookName = document.getElementById('book_name');
+const autName = document.getElementById('auther_name');
+const addBtn = document.getElementById('addButton');
+
+// const bookSection = document.getElementById("book_section");
```

Comment:
- [ ] Kindly remove all commented-out, un used, or unwanted pieces of code from your code base. It makes your code base look messy and cluttered.